Tests/Testo/Invoice/Helpers/PublicDocumentAssetHelperTest.php

```
@@ -42,18 +51,18 @@ public function resolveAsksForTheExactBootstrapAndIconsAssetsAndInlinesCustomPdf
 
-        $tmpName = tempnam(sys_get_temp_dir(), 'custom-pdf-');
-        if ($tmpName === false) {
-            Assert::fail('tempnam() failed to allocate a temp file');
-        }
-        $cssPath = $tmpName . '.css';
+        // A fake '@root' with the real nested path the helper actually
+        // reads from -- not just a bare file handed straight to Aliases::get()
+        // -- so this test would have caught the '@invoice' mistake.
+        $fakeRoot = sys_get_temp_dir() . '/public-document-asset-helper-' . uniqid();
+        $cssDir = $fakeRoot . '/src/Invoice/Asset/core/css';
+        Assert::true(mkdir($cssDir, 0777, true));
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Testo\Assert' in method 'resolveAsksForTheExactBootstrapAndIconsAssetsAndInlinesCustomPdfCss'. (undefined)

(StaticAccess)